bindings/postgres/postgres.go

```diff
@@ -0,0 +1,176 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+package postgres
+
+import (
+	"context"
+	"encoding/json"
+	"strconv"
+	"time"
+
+	"github.com/dapr/components-contrib/bindings"
+	"github.com/dapr/dapr/pkg/logger"
+	"github.com/pkg/errors"
+
+	"github.com/jackc/pgx/v4/pgxpool"
+)
+
+// List of operations.
+const (
+	ExecOperation  bindings.OperationKind = "exec"
```

Comment:
these constants need to start with lower case. because these will be used in this package.

Reply:
fixed